### PR TITLE
US17_sendmail

### PR DESCRIPTION
**Pull request by khadimb**

### CODE REVIEW

**Review by Vannou28 — DISMISSED** [review]

File: src/Controller/ContactController.php [hunk 1]
```diff
@@ -13,72 +15,40 @@ class ContactController extends AbstractController
      * @throws \Twig\Error\SyntaxError
      */
 
-    private array $errors = [];
+
+    protected array $errors = [];
```
Comment (LucyMerlier) [inline src/Controller/ContactController.php:19]: why make it protected? do you plan on having children classes of this controller?

File: src/Controller/ContactController.php [hunk 2]
```diff
@@ -13,72 +15,40 @@ class ContactController extends AbstractController
...
-            $this->checkFilterValidateEmail($data['email'], 'L\'email saisi n\'est pas valable');
+            $validation->wordIsInArray($data['subject'], $subjects, 'Le sujet saisi n\'est pas valable');
+            $validation->emailFilterValidate($data['email'], 'L\'email saisi n\'est pas valable');
 
-            if (empty($this->getErrors())) {
+            if (empty($validation->getErrors())) {
+                    $message = 'Voici avez reçu un nouveau message de la part de ' . $data['name'] .
+                    ', vous pouvez le joindre à ce numéro : ' . $data['tel'] . ' ou part mail : ' . $data['email'] .
+                     'Son message est : ' . $data['message'];
+                    mail("projetvoyance@gmail.com", $data['subject'], $message);
```
Comment (LucyMerlier) [inline src/Controller/ContactController.php:47]: check your message again, i don't think it's clear who it's supposed to be destined to

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**